.vib/opencart/cypress/cypress/fixtures/products.json

```diff
@@ -0,0 +1,7 @@
+{
+  "newProduct": {
+    "name": "Apple iWatch",
+    "metatagTitle": "appleiwatch",
+    "model": "Apple"
```

Comment:
I recommend you use made-up products/brands. I don't know the legal behind using real ones, so inventing them rules out that issue.

Reply:
I've replaced this fruit brand with an invented fruit brand 🍒 .

Reply:
🚀